ts/sagas/user/userMetadata.ts

```diff
@@ -45,6 +46,11 @@ export function* fetchUserMetadata(
     }
 
     if (response.value.status !== 200) {
+      if (response.value.status === 204) {
+        // Return an empty object
```

Comment:
```suggestion
        // Return an empty object cause profile has no metadata yet (204 === No Content)
```
Could be useful a more detailed description